packages/sdk/src/sdk/api/remix-contests/RemixContestApi.ts

```diff
@@ -0,0 +1,90 @@
+import snakecaseKeys from 'snakecase-keys'
+
+import { LoggerService } from '../../services'
+import type { StorageService } from '../../services'
+import {
+  EntityManagerService,
+  EntityType,
+  Action
+} from '../../services/EntityManager/types'
+import { decodeHashId } from '../../utils/hashId'
+import {
+  Configuration,
+  RemixContestsApi as GeneratedRemixContestsApi
+} from '../generated/default'
+
+type RemixContestMetadata = {
+  eventId: number
+  trackId: number
+  endDate: string // ISO format date string
+}
+
+type ExtendedConfiguration = Configuration & {
```

Comment:
Unless there was a change recently, the APIs don't take userId as an initialization parameter. It's passed as part of the function parameters for each API call.